srcpkgs/python3-pgzero/template

```diff
@@ -1,15 +1,15 @@
 # Template file for 'python3-pgzero'
 pkgname=python3-pgzero
-version=1.2
-revision=3
+version=1.2.1
+revision=1
 wrksrc="pgzero-${version}"
 build_style=python3-module
-pycompile_module="pgzero pgzrun.py"
 hostmakedepends="python3-setuptools"
-depends="python3-setuptools python3-pygame python3-numpy"
+depends="python3-setuptools python3-pygame>=2.0.0 python3-numpy"
```

Comment:
I don't think we need to specify the version here, since it will come in a single update to the dependency chain.

Reply:
Ok, I have removed this.